routes/event.js

```diff
@@ -315,6 +358,70 @@ function sendUpdateEmail(event) {
 	});
 }
 
+function addGuestDetails(event, details, callback) {
+	var guestDetails = details;
+
+	for (var i = 0; i < event.invitees.length; i++)
+	{
+		if (event.invitees[i].state == 'Attending')
+		{
+			guestDetails = guestDetails + 'Guest Email: ' + event.invitees[i].email + '\nGuest Access Requirements: ' + event.invitees[i].accessRequirements + '\nGuest Dietary Restrictions: ' + event.invitees[i].dietaryRestrictions + '\n\n';
+		}
+	}
+
+	callback(guestDetails);
+}
+
+function addTableDetails(event, guestDetails, callback) {
+	var guestAndTableDetails = guestDetails;
+
+	tableMapper.findTablesByEventId(event.event_id, function(err,table) {
+		if (err) {
+			req.flash('err', err);
+		}
+		else if (table != null) {
+			for (var x = 0; x < table.length; x++) {
+				guestAndTableDetails = guestAndTableDetails + 'Table: ' + table[x].tableNumber + '\n';
+
+				for (var y=0; y < table[x].tableLabels.length; y++) {		
+					guestAndTableDetails = guestAndTableDetails + table[x].tableLabels[y];
+							
+					if (y != table[x].tableLabels.length-1) {
+						guestAndTableDetails = guestAndTableDetails + ', ';
+					}
+				}
+
+				guestAndTableDetails = guestAndTableDetails + '\n';
+			
+				for(var z=0; z < table[x].seatLabels.length; z++) {
+					guestAndTableDetails = guestAndTableDetails + 'Seat ' + (z + 1) + ': ' + table[x].seatLabels[z] + '\n';
+				}
+
+				guestAndTableDetails = guestAndTableDetails + '\n';
+			}
+		}
+		else
+		{
+			guestAndTableDetails = guestAndTableDetails + 'No tables have been allocated for the event.';
```

Comment:
This condition doesnt seem to get run for me. I think findTables returns array so will always be not null. Need to check not null and the length